code-of-conduct.md

```diff
@@ -0,0 +1,18 @@
+## Kubernetes Community Code of Conduct
+
+The Kubernetes open source community and project is a professional community dedicated to advancing the
+state of the art in container cluster management.  Participants in the Kubernetes community are
+expected to behave according to professional standards and in accordance with their employer's policies
+on appropriate workplace behavior.
+
+While participating in the Kubernetes open source project, Kubernetes events, or related social networking
+opportunities, participants should not engage in discriminatory or offensive speech or actions regarding
+gender, sexuality, race, or religion. At Kubernetes events or meet-ups, speakers should be especially
+aware of these concerns.
```

Comment:
I'd love to expand the list to something like "regarding (and not limited to) level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, or religion"